payments/trustly/index.md

```diff
@@ -53,6 +53,47 @@ operations][after-payment] and the [payment resource][payment-resource].
 
 The displayed languages that we currently support is English (`en-us`), Norwegian (`nb-no`), Swedish (`sv-se`), and Finish (`fi-fi`).
 
+## Banks
+
+Trustly is supported by the following Finnish and Swedish banks as of August 14.
+2020.
+
+### Finland {% flag fi %}
+
+{:.table .table-striped}
+| :------------ | :------------------- | :-------------------- |
+| Bank   | Supported in eCom as of August 2020  |  Instant Credit Notification  |
```

Comment:
```suggestion
| Bank   | Supported in eCom as of August 2020  |  Instant Credit Notification  |
| :------------ | :------------------- | :-------------------- |
```

Reply:
I'll try! I goofed something and can't get the localhost to start, so this check-in was blind.